Documentation/gettingstarted/terminology.rst

```diff
@@ -243,8 +243,7 @@ identities are prefixed with the string ``reserved:``.
 
    Cilium used to include both the local and all remote hosts in the
    ``reserved:host`` identity. This is still the default option unless a recent
-   default ConfigMap is used. The remote-node identity can be enabled via
-   the option ``enable-remote-node-identity``.
+   default ConfigMap is used. The remote-node identity is enabled by default since Cilium v1.16.
```

Comment:
It's enabled by default since Cilium v1.7.

I think we should remove the whole paragraph however, it doesn't make sense anymore really.

Reply:
This was marked as resolve but the paragraph is still here?